src/app/component/ProductCard/ProductCard.component.js

```diff
@@ -283,16 +335,18 @@ export class ProductCard extends PureComponent {
         return (
             this.renderCardLinkWrapper((
                 <>
-                    <figure block="ProductCard" elem="Figure">
-                        { this.renderPicture() }
-                    </figure>
-                    <div block="ProductCard" elem="Content">
+                    <div block="ProductCard" elem="Figure-Review">
```

Comment:
This does not seem right, consider not using `_` and `-` in blocks, mods, elems.

Reply:
Fixed